test/run_each.sh

```diff
@@ -34,7 +34,7 @@ runTest () {
 runTest 'Help/test/*.js,Help/js/*.js'
 runTest 'History/js/*.js,History/test/*.js'
 runTest 'Login/js/*.js,Login/test/*.js'
-runTest 'Pages/js/options.js,Pages/js/tests/*.js'
+runTest 'Pages/js/*.js,Pages/js/tests/*.js'
```

Comment:
This will test both the options page and the first run page. I think they likely need to be tested separately at the moment.